cra-tf-validate-ignore-goals.json

```diff
@@ -36,6 +36,18 @@
             "description": "Check whether Flow Logs for VPC are enabled",
             "ignore_reason": "Flow logs are enabled, so seems to be CRA bug. Tracking in https://github.ibm.com/oneibmcloud/CD-CRA/issues/1779",
             "is_valid": false
+        },
+        {
```

Comment:
Are those new warnings when moving to the module? There is no difference in this area between what the module does and the original version, so surprised.

Reply:
Do we know more on this?

Reply:
Looks like `3000408` is no longer failing using latest version of CRA so can be removed from this

Reply:
Looks like `3000452`, `3000805` and `3000804` are also not failing either and should be removed

Reply:
PS: Soon we will move to running CRA with --sccv2 profile, so I imagine the contents of this JSON will change